airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -188,7 +188,7 @@ def _serialize_exception(
 
     @property
     def raise_exception_on_missing_stream(self) -> bool:
-        return True
+        return False
```

Comment:
It seems like this will break the behavior for some connectors that expects this to break. Do we have a migration plan?

Reply:
I'm ok to leave as is and create another PRs to change default behavior for the following connectors:
![image](https://github.com/user-attachments/assets/2e78176f-c01a-4ade-9e9e-86463d95b947)
- shopify
- facebook0marketing
- google-ads
- google-analytics-data-api